spec/services/submission_service_spec.rb

```diff
@@ -479,6 +480,50 @@
       expect(submission.rejection_reason).to eq nil
     end
 
+    context 'Braze integration' do
```

Comment:
If #1254 is merged before, we need to update this.

Reply:
It was merged a few minutes ago 🙁